pkg/agent/nodeportlocal/util/parse_port.go

```diff
@@ -41,3 +45,19 @@ func ParsePortsRange(portRangeConfig string) (start, end int, err error) {
 
 	return start, end, nil
 }
+
+// BuildPortProto creates a single string using port and protocol separated by a delimiter.
+func BuildPortProto(port, protocol string) string {
+	return port + delim + strings.ToLower(protocol)
+}
```

Comment:
looking at how the function is used, and for symmetry with `ParsePortProto`, it seems to me that it would make more sense for the `port` parameter to be of type `int`?